frontend/src/scenes/experiments/ExperimentResult.tsx

```diff
@@ -37,181 +37,197 @@ export function ExperimentResult(): JSX.Element {
+                                                ) : (
+                                                    <div className="space-x-1">
+                                                        <span>
+                                                            <b>Conversion rate:</b>{' '}
+                                                        </span>
+                                                        <span>{conversionRateForVariant(variant)}%</span>
+                                                    </div>
+                                                )}
+                                                <LemonProgress
+                                                    percent={Number(
+                                                        (experimentResults.probability[variant] * 100).toFixed(1)
                                                     )}
+                                                    strokeColor={getSeriesColor(
+                                                        getIndexForVariant(variant, experimentInsightType)
+                                                    )}
+                                                />
+                                                <div>
+                                                    Probability that this variant is the best:{' '}
+                                                    <b>{(experimentResults.probability[variant] * 100).toFixed(1)}%</b>
                                                 </div>
-                                                <div className="flex">
-                                                    <b className="pr-1">Exposure:</b>{' '}
-                                                    {exposureCountDataForVariant(variant)}
-                                                </div>
-                                            </>
-                                        ) : (
-                                            <div className="space-x-1">
-                                                <span>
-                                                    <b>Conversion rate:</b>{' '}
-                                                </span>
-                                                <span>{conversionRateForVariant(variant)}%</span>
                                             </div>
-                                        )}
-                                        <LemonProgress
-                                            percent={Number((experimentResults.probability[variant] * 100).toFixed(1))}
-                                            strokeColor={getSeriesColor(
-                                                getIndexForVariant(variant, experimentInsightType)
+                                        ))
+                                }
+                            </div>
+                        )
+                    ) : (
+                        !experimentResults && experimentResultsLoading && <LoadingState />
```

Comment:
github diffs make this hard to read, but do we need this here at all, given you now have it at the top level on L40-45 ?